sapi/fuzzer/fuzzer-tracing-jit.c

```diff
@@ -62,12 +62,18 @@ int LLVMFuzzerInitialize(int *argc, char ***argv) {
 		"zend_extension=%s\n"
 		"opcache.validate_timestamps=0\n"
 		"opcache.file_update_protection=0\n"
-		"opcache.jit_buffer_size=256M\n"
+		"opcache.memory_consumption=1024\n"
+		"opcache.jit_buffer_size=1024M\n"
 		"opcache.jit_hot_func=1\n"
 		"opcache.jit_hot_loop=1\n"
 		"opcache.jit_hot_return=1\n"
 		"opcache.jit_hot_side_exit=1\n"
-		"opcache.jit_max_root_traces=32768",
+		"opcache.jit_max_root_traces=100000\n"
+		"opcache.jit_max_side_traces=100000\n"
+		"opcache.jit_max_exit_counters=100000\n"
+		"opcache.jit_blacklist_root_trace=255\n"
+		"opcache.jit_blacklist_side_trace=255\n"
```

Comment:
Why do you increase "blacklist" settings?

Reply:
@dstogov Could you clarify a bit? Looking at the logic in zend_jit_trace_is_bad_root, it seemed to me that the higher the blacklist value, the less likely it is for a given trace to be blacklisted; by increasing the value, if the same trace errors out N-2 times, but then at the N-1th time gets compiled successfully, and N is the maximum allowed value (255), this should lead to an overall increase in the number of compiled traces; am I missing anything, and if yes, mind also updating the docs to clarify it a bit? :)

Reply:
Usually, blacklisting occurs when the same error or tracing condition repeats again and again.
It's hardly ever the same trace is going to be compiled successfully after a few failed attempts.
So at least you reduce testing of blacklisting, that actually produces JIT code for de-optimization.
I would recommend to keep the default values.

Reply:
OK then, reverted!